src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  actions.forEach(howTransform => {
```

Comment:
```suggestion
  actions.forEach(action => {
```
![image](https://github.com/mate-academy/js_stateful-object/assets/96409563/8d23c37a-cc50-4f87-8767-3cd18fad4e67)